security/oidc-client-mutual-tls/src/test/java/io/quarkus/ts/security/oidcclient/mtls/AuthenticationTestSuite.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkus.ts.security.oidcclient.mtls;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+
+public interface AuthenticationTestSuite {
+
+    String RESOURCE_PATH = "/ping";
+    String NORMAL_USER = "test-normal-user";
+
+    String getToken(String username);
+
+    @Test
+    default void verifyAuthenticated() {
+        given()
+                .when()
+                .auth().oauth2(getToken(NORMAL_USER))
+                .get(RESOURCE_PATH)
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .body(equalTo(NORMAL_USER + "@gmail.com"));
+    }
+
+    /**
+     * Ensures that {@link AuthenticationTestSuite#verifyAuthenticated()} result make sense.
```

Comment:
This commend should be reviewed

Reply:
I removed the comment and added a new test. Now I think context (method name in context of its class) shall clarify method itself.